_posts/2015-12-24-two-column.markdown

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: 2 column Blog with Video
+date: 12/24/15
```

Comment:
Hello and thanks for the PR! Please use YYYY-MM-DD HH:MM:SS date format. If time is not available then please set it as 00:00:00.